synth.py

```diff
@@ -30,7 +30,7 @@
     },
     proto_path='/google/monitoring/dashboard/v1',
     version=version)
-s.copy(library, excludes=['README.md'])
+s.copy(library, excludes=['README.md', 'package.json'])
```

Comment:
Please remove line 41 below (`rm -rf` of the `fixtures` folder).
Also, you'll need to add `npx compileProtos src`.

Also, to make fixtures have the correct package name, the `generator_args` above should have `'package-name': '@google-cloud/monitoring-dashboard'`.